### PR TITLE
Add a document for Command Executor

### PR DESCRIPTION
**Pull request by yoyama**
Add a doc for Command Executor

### CODE REVIEW

File: digdag-docs/src/conf.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ['recommonmark']
+extensions = ['recommonmark', 'sphinx_markdown_tables']
```
Comment (yoyama) [inline digdag-docs/src/conf.py:37]: Need for table in markdown.

File: digdag-docs/requirements.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
 sphinx
 recommonmark
 sphinx_rtd_theme
+sphinx-markdown-tables
```
Comment (yoyama) [inline digdag-docs/requirements.txt:4]: Need for table in markdown.

**Comment by yoyama** [comment]
This document does not cover the all. You can add/update document with new PR.

File: digdag-docs/src/command_executor.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
+# Command Executor
+`Command Executor` provides a functionality to run operators in various environment like Docker.
+
+`sh>`, `py>` `rb>` support Command Executor.
```
Comment (serihiro) [inline digdag-docs/src/command_executor.md:4]:
```suggestion
`sh>`, `py>`, and `rb>` support Command Executor.
```

File: digdag-docs/src/command_executor.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+agent.command_executor.ecs.digdag-test.region = us-east-1
+agent.command_executor.ecs.digdag-test.subnets = subnet-NNNNN
+agent.command_executor.ecs.digdag-test.max_retries = 3
+
+agent.command_executor.ecs.temporal_storage.type = s3
+agent.command_executor.ecs.temporal_storage.s3.bucket = <Bucket>
+agent.command_executor.ecs.temporal_storage.s3.endpoint = s3.amazonaws.com
+agent.command_executor.ecs.temporal_storage.s3.credentials.access-key-id = <ACCESS KEY>
+agent.command_executor.ecs.temporal_storage.s3.credentials.secret-access-key = <SECRET KEY>
+```
```
Comment (serihiro) [inline digdag-docs/src/command_executor.md:35]: Regarding `access-key-id` and `secret-access-key`, we should mention about the fallback to commonly applied values (default vlaues) for all ECS clusters.

https://github.com/treasure-data/digdag/blob/cb35cf213c5afe896f3b385ad1ceb78ff12f6dec/digdag-standards/src/main/java/io/digdag/standards/command/ecs/EcsClientConfig.java#L78-L88

**Review by serihiro — COMMENTED** [review]
@yoyama 
I left minor comments. Could you take a look?

File: digdag-docs/src/command_executor.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+    image: "digdag/digdag-python:3.7"
+
++task1:
+  py>: ...
+```
+
+You need to set a tag `digdag.docker.image` in the task definition.
+ECS Command Executor try to search the tagged task definition.
+
+(This way lists and check all task definitions until found and take long time to run the task. See issue #1488)
```
Comment (szyn) [inline digdag-docs/src/command_executor.md:86]:
```suggestion
(This way lists and check all task definitions until found and take long time to run the task. See issue [#1488](https://github.com/treasure-data/digdag/issues/1488))
```

File: digdag-docs/src/command_executor.md (hunk 4 above)
Comment (yoyama) [reply digdag-docs/src/command_executor.md:35]: @serihiro Thank you for your review.
The description focuses on the minimum/basic configuration to start ECS Command Executor.
If we describes about default `__default_config__`, it is better to add a sub-section "multiple cluster setting".
But I don't have enough time to write them and skip it. 
What do you think it ?
If you think we need to add them, you can create additional PR for it.

Comment (serihiro) [reply digdag-docs/src/command_executor.md:35]: Then I will add it to another PR later. Thanks.

**Review by serihiro — APPROVED** [review]

**Pull request merged**